Tests/KituraTests/KituraTest.swift

```diff
@@ -189,6 +302,34 @@ class KituraTest: XCTestCase {
     func expectation(line: Int, index: Int) -> XCTestExpectation {
         return self.expectation(description: "\(type(of: self)):\(line)[\(index)](ssl:\(useSSL))")
     }
+
+    // Generates a unique temporary file path suitable for use as a Unix domain socket.
+    // On Linux, a path is returned within /tmp
+    // On MacOS, a path is returned within /var/folders
+    func uniqueTemporaryFilePath() -> String {
```

Comment:
Can we use Foundation on Linux too?

Reply:
FileManager.temporaryDirectory wasn't implemented in Swift 4.0.3